plugins/services/src/js/containers/service-configuration/__tests__/ServiceConfigurationContainer-test.js

```diff
@@ -20,14 +20,36 @@ describe("ServiceConfigurationContainer", function() {
       lastConfigChangeAt: "2016-03-22T10:46:07.354Z",
       lastScalingAt: "2016-03-22T10:46:07.354Z"
     },
-    versions: new Map([["2016-05-02T16:07:32.583Z", { id: "/test" }]])
+    versions: new Map([
+      [
+        "2016-05-02T16:07:32.583Z",
+        {
+          id: "/test",
+          container: {
+            volumes: [
+              {
+                containerPath: "/mnt/volume",
+                external: {
+                  name: "someVolume",
+                  provider: "dvdi",
+                  options: {
+                    "dvdi/driver": "rexray"
+                  }
+                },
+                mode: "RW"
+              }
+            ]
+          }
+        }
+      ]
+    ])
   });
 
   beforeEach(function() {
     this.container = global.document.createElement("div");
     this.instance = ReactDOM.render(
       <ServiceConfigurationContainer
-        onEditClick={function() {}}
+        onEditClick={() => {}}
```

Comment:
Nice 😏

Reply:
so instead of creating a function that does nothing we're creating a function that does nothing + binding a context for no particular reason 😼